lib/connections/relay_listen.go

```diff
@@ -71,18 +74,18 @@ func (t *relayListener) Serve() {
 
 			conn, err := client.JoinSession(inv)
 			if err != nil {
-				l.Infoln("Joining relay session (BEP/relay):", err)
+				l.Debugln("Listen (BEP/relay): failed to join session:", err)
```

Comment:
Should we really hide these? It's one of the only ways to debug the issues that are happening now.

Reply:
That was what I was asking originally. I was confused by what looked like randomly chosen info or debug levels with or without context (e.g. SetTCPOptions was info but without context, SetTrafficClass was debug but with context). I can't really judge their relevance, so pointers whether a log statement should be info or debug like this one are appreciated.

Reply:
Other errors that are returned by dial functions etc are logged a layer up, in the loop, so it's ok, yet this is the only place that reports relay issues, so it's not cool to hide it.

Reply:
The dial logging was only on debug, I changed that to info and removed some redundant debug log statements in the dial functions themselves.